demo-shell/src/app/components/process-service/process-service.component.html

```diff
@@ -20,6 +20,7 @@
                                 [filterParam]="{name:'MY tasks'}"
                                 [appId]="appId"
                                 [hasIcon]="false"
+                                [landingFilterId]="7"
```

Comment:
As a developer, I'd like that fetch that id value from the query string. 
In this way will be easy also for the QA to test it

Result
http://localhost:3000/activiti/apps/1/tasks?landingFilterId=7